README.md

```diff
@@ -194,12 +195,12 @@ This testing is generally done by a developer
 
 To run components tests run the following commands
 
-    make tester-build
+    make tester-build mock-dos-db-setup
```

Comment:
Just need some text to explain the `mock-dos-db-setup` target need the developer to be mfa'ed and why.

Reply:
Updated with comment to sign into non-prod